spec/files/skip_zip_results_osw/measures/IncreaseRoofRValue/measure.rb

```diff
@@ -0,0 +1,299 @@
+#start the measure
+class IncreaseInsulationRValueForRoofsByPercentage < OpenStudio::Ruleset::ModelUserScript
```

Comment:
yikes, we have some old files in here. I'm sure we could upgrade these to be `OpenStudio::Measure`, but let's do that in a different PR. Also, the dencity measures should be deprecated.